Brewfile

```diff
@@ -0,0 +1,2 @@
+brew "tesseract"
+cask "libreoffice"
```

Comment:
```suggestion
cask "libreoffice"
brew "wkhtmltopdf"
```

Following https://github.com/guardian/giant/pull/65 we'll need this too

Reply:
nice spot